symbol_exporter/ast_symbol_extractor.py

```diff
@@ -34,9 +42,9 @@ def __lt__(self, other):
 
 class SymbolType(OrderedEnum):
     STAR_IMPORT = 1
-    RELATIVE_STAR_IMPORT = 2
-    IMPORT = 3
-    RELATIVE_IMPORT = 4
+    RELATIVE_STAR_IMPORT = 10
```

Comment:
You might add a small docstring or comment to note that the ordering is the clobbering order of precedence, larger numbers mean the type will most likely not be overridden.

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
done